src/Microsoft.ML.PipelineInference/AutoMlUtils.cs

```diff
@@ -15,21 +15,34 @@ namespace Microsoft.ML.Runtime.PipelineInference
 {
     public static class AutoMlUtils
     {
-        public static AutoInference.RunSummary ExtractRunSummary(IHostEnvironment env, IDataView data, string metricColumnName)
+        public static double ExtractValueFromIDV(IHostEnvironment env, IDataView result, string columnName)
         {
-            double metricValue = 0;
-            int numRows = 0;
-            var schema = data.Schema;
-            schema.TryGetColumnIndex(metricColumnName, out var metricCol);
+            Contracts.CheckValue(env, nameof(env));
+            env.CheckValue(result, nameof(result));
+            env.CheckNonEmpty(columnName, nameof(columnName));
 
-            using (var cursor = data.GetRowCursor(col => col == metricCol))
+            double outputValue = 0;
+            var schema = result.Schema;
+            if (!schema.TryGetColumnIndex(columnName, out var metricCol))
+                throw env.ExceptParam($"Schema does not contain column: {columnName}");
```

Comment:
>xceptParam($"Schema does not contain column: {columnName}"); [](start = 27, length = 60)

Similar here, misuse of `ExceptParam`'s first arg. I guess it's either `result` or `columnName`.

Reply:
Good catch. In switching over to ExceptParam I didn't change to reflect the new signature. Error on my part.